src/sentry/models/organization.py

```diff
@@ -194,7 +194,7 @@ def get_audit_log_data(self):
             'id': self.id,
             'slug': self.slug,
             'name': self.name,
-            'status': self.status,
+            'status': int(self.status),
```

Comment:
AuditLogEntry uses pickle at the moment

We should probably just rewrite that thing to remove the type(int) column (change to str), and change data to be a JSON field.